recipes/galaxy-ml/meta.yaml

```diff
@@ -11,8 +11,7 @@ source:
 
 build:
   number: 0
-  skip: True
```

Comment:
I'm new to bioconda recipes, but this one failed previously without `skip: True`. @dpryan79

Reply:
With `skip: True` you're telling the build system to not build the package.

Reply:
Are you at GCC? If so, please remind me to have a look at this PR tomorrow so we can get it building and made available.

Reply:
Or if you're in Jeremy's group just ask him to remind me :)

Reply:
I'm not, unfortunately. I can ping you though if the problem is still there.

Reply:
Sounds cool.